_source/_mhe-nmds-data-entry-tool/2019-2020.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: MHE NMDS Data Entry Tool for 2019-2020
+direction: MHE NMDS Data Entry Tool
+directory:
+  - url: /mhe-nmds-data-entry-tool/index.html
+    name: MHE NMDS Data Entry Tool
+last_modified: 1
+current_release: 1
+---
+
+The [2019-2020 version of the MHE MDS Data Entry Tool (zip format, 573 kB)][interface-href] is available for download.
+If you are downloading this tool for the first time you will also require the ["back end" database (zip format, 66 kB)][emptydata-href]. Separating the user interface from the data store in this manner allows us to update the interface without overwriting your data.
+These files are distributed in zipped form. Extract them both to the same folder on your computer and open the MHE-NMDS-2019-2020-Interface.mdb file with Microsoft Access.
+**NOTE** If you have entered any data **DO NOT** extract the back end database (mhe-nmds-2019-2020-empty-data.zip) file into the same directory as your old MHE NMDS database or you will lose your data.
+
+The [user guide (pdf format, 1.2 MB)][userguide-href] is available for download.
+## Download files
+* [MHE NMDS Data Entry Tool - Data (Empty) 2019-2020 (zip format, 66 kB)][emptydata-href]
+* [MHE NMDS Data Entry Tool - Interface (zip format, 573 kB)][interface-href]
+* [MHE NMDS Data Entry Tool User Guide (pdf format, 1.2 MB)][userguide-href]
+
+[interface-href]: /site/assets/files/1037/MHE-NMDS-2019-2020-interface.zip
+[emptydata-href]: /site/assets/files/1037/MHE-NMDS-2019-2020-empty-data.zip
+[userguide-href]: /site/assets/files/1037/MHE-NMDS-2019-2020-DE-Tool-User-Guide.pdf
```

Comment:
See earlier comment.

Reply:
no reason at all, I have very little idea what I'm doing, am just trying to copy what has been done in the past - I've removed those directories and updated the file paths